cinema_service/urls.py

```diff
@@ -10,7 +10,7 @@
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-    path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("z", include("cinema.urls", namespace="cinema")),
```

Comment:
There seems to be a typo in the path for the cinema app. The path is set to 'z' instead of something more descriptive like 'api/cinema/'. Consider correcting this to ensure clarity and consistency with other API paths.